### PR TITLE
Added storing MapBox token with dotenv

### PR DESCRIPTION
**Pull request by jsantos42**
Added a Zoey box suggesting an alternative and safer way of storing the MapBox token, using dotenv.

### CODE REVIEW

File: src/markdown/tutorial/part-1/07-reusable-components.md [hunk 1]
```diff
@@ -56,6 +56,16 @@ yarn ember test
 git add config/environment.js
 ```
 
+> Zoey says...
+>
+> The above described way of storing your MapBox token will expose it on the repository once you upload it. There is a safer way to store your public token, using dotenv:
```
Comment (jaredgalanis) [inline src/markdown/tutorial/part-1/07-reusable-components.md:61]: @jsantos42, thank you for your contribution! 

We agree that users should not be committing access tokens to version control, but we think that identifying the solution might over complicate things here for new users. 

Can you please try to make this a more generalized suggestion about not committing access tokens to git and pushing them to Github? 👍

**Review by jaredgalanis — CHANGES_REQUESTED** [review]

File: src/markdown/tutorial/part-1/07-reusable-components.md (hunk 1 above)
Comment (chancancode) [reply src/markdown/tutorial/part-1/07-reusable-components.md:61]: Yeah I think this is too much for the purpose of the tutorial. I also don't think committing this particular token to Git is actually a big deal, it's not really a secret, it's more like a username than a password – the full token is going to be in every API request in plaintext _served to your end-users_ anyway, leaking it via version control is the last thing you have to worry about here.

Comment (jsantos42) [reply src/markdown/tutorial/part-1/07-reusable-components.md]: @jaredgalanis is it better? I also corrected some linting errors.

Comment (jsantos42) [reply src/markdown/tutorial/part-1/07-reusable-components.md]: Hey guys! Any update here?

Comment (MinThaMie) [reply src/markdown/tutorial/part-1/07-reusable-components.md]: I just see your change :) This looks great

File: src/markdown/tutorial/part-1/07-reusable-components.md [hunk 2]
```diff
@@ -56,6 +56,11 @@ yarn ember test
 git add config/environment.js
 ```
 
+> Zoey says...
+>
+> The above described way of storing your Mapbox token will expose it on the repository once you upload it. There are safer ways of storing your public token (eg, using [dotenv](https://www.npmjs.com/package/ember-cli-dotenv)), but that is beyond the scope of this tutorial.
```
Comment (MinThaMie) [inline src/markdown/tutorial/part-1/07-reusable-components.md:61]:
```suggestion
> The above described way of storing your Mapbox token will expose it in the repository once you upload it. There are safer ways of storing your public token (eg, using [dotenv](https://www.npmjs.com/package/ember-cli-dotenv)), but that is beyond the scope of this tutorial.
```

**Review by MinThaMie — CHANGES_REQUESTED** [review]
Looks good, just a minor syntax thing :)

File: src/markdown/tutorial/part-1/07-reusable-components.md (hunk 1 above)
Comment (jsantos42) [reply src/markdown/tutorial/part-1/07-reusable-components.md]: Ups! I missed that one, thank you for the grammar correction @MinThaMie!

**Review by MinThaMie — APPROVED** [review]
Thanks :)

**Pull request closed without merging**